open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/files/egeria-ui/application.properties

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project.
+title = 'Open Metadata Platform:|Find the right data with governance. (dev)'
+description = 'Have a question?||Visit https://github.com/odpi/egeria .@@What is Open Metadata?||Start your customer journey with this link to know what is Open Metadata and how can you use this applicaiton.@@Have more cool ideas?||Feel free to let us know your ideas and business challenges so we can make it better.'
```

Comment:
Let's revert this change with the deployment and bring the fix from master, please charry-pick: https://github.com/odpi/egeria/pull/5245/commits/1b7b925dbf4f870595475f62ba773ff75efeaf4b

Reply:
Yes agreed. Will do. Thanks for proposing the fix